content/docs/04-clusters/02-data-center/02-openstack.md

```diff
@@ -361,16 +362,239 @@ The following prerequisites must be met before deploying a Kubernetes clusters i
 
 # Installing Private Cloud Gateway - OpenStack
 
+
+Use the following steps to install a PCG cluster in your OpenStack environment. You can use the [Palette CLI](/palette-cli) or the PCG Installer Image to deploy a PCG cluster. Review the prerequisites for each option to help you identify the correct install method.
+
+
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Palette CLI" key="palette-cli">
+
+
+## Prerequisites
+
+
+The following system prerequisites are required to install an OpenStack PCG.
+
+- Palette version 4.0.X or greater.
+
+
+- A Palette API key. Refer to the [Create API Key](/user-management/user-authentication#apikey) page for guidance.
+
+
+- Download the Palette CLI from the [Downloads](/spectro-downloads#palettecli) page and install the CLI. Refer to the [Palette CLI Install](/palette-cli/install-palette-cli) guide to learn more.
+
+
+- PCG IP address requirements: <br />
+
+    - Depending on topology, either one IP address for a single-node PCG or three IP addresses for a three-node HA PCG.
+
+    - One IP address for the Kubernetes control plane.
+
+    - One additional Kubernetes control plane IP address for rolling upgrades.
+
+
+- A Linux x86-64 host with the Docker daemon installed.
+
+
+- Sufficient available IP addresses within the configured OpenStack subnets.
+
+
+## Install PCG
+
+
+1. In an x86 Linux host, open up a terminal session.
+
+
+
+2. Use the Palette CLI `login` command to authenticate the CLI with Palette. When prompted, enter the information listed in the following table
+
+    <br />
+
+    ```shell
+    palette login
+    ```
+
+    <br />
+
+    |**Parameter**       | **Description**|
+    |:-----------------------------|---------------|
+    |**Spectro Cloud Console** |Enter the Palette endpoint URL. When using the Palette SaaS service, enter ``https://console.spectrocloud.com``. When using a self-hosted instance of Palette, enter the URL for that instance. |
+    |**Allow Insecure Connection** |Enabling this option bypasses x509 verification. Enter `y` if you are using a self-hosted Palette instance with self-signed TLS certificates. Otherwise, enter `n`.|
```

Comment:
@karl-cardenas-coding The description of the **Allow Insecure Connection** parameter in this doc and the one in the VMware doc differ. Let's choose the best one and use it for both.

I notice this same parameter appears in two tables within the same doc, as it also does in the VMware doc - should it? The parenthetical _(bypass ...)_ remark in the VMware doc shouldn't be in the Parameter column in the VMware doc.

![CleanShot 2023-07-18 at 15 37 44](https://github.com/spectrocloud/librarium/assets/117382432/f57d51a2-aad6-4b3d-9798-35b13e86dd3b)

Reply:
Fixed